src/components/CohortBuilder/SqonBuilder/index.js

```diff
@@ -1,25 +1,26 @@
 import React from 'react';
-import { compose } from 'recompose';
-import omit from 'lodash/omit';
 import memoize from 'lodash/memoize';
-import { injectState } from 'freactal';
 import AdvancedSqonBuilder from '@kfarranger/components/dist/AdvancedSqonBuilder';
 import ExtendedMappingProvider from '@kfarranger/components/dist/utils/ExtendedMappingProvider';
-
-import { withApi } from 'services/api';
-import { trackUserInteraction, TRACKING_EVENTS } from 'services/analyticsTracking';
+import { TRACKING_EVENTS, trackUserInteraction } from 'services/analyticsTracking';
 import { arrangerProjectId } from 'common/injectGlobals';
 import { ARRANGER_API_PARTICIPANT_INDEX_NAME } from '../common';
 import { FieldFilterContainer } from '../FieldFilterContainer';
-import { ModalContentSection } from '../ModalContentSection.tsx';
 import { SQONdiff } from '../../Utils';
-import { ModalFooter } from 'components/Modal';
-
 import './SqonBuilder.css';
+import { Button, Modal, Typography } from 'antd';
```

Comment:
I would keep the css import at they end and not mix it with the .js import

Reply:
good to know